scripts/migrations.lua

```diff
@@ -223,6 +224,9 @@ migrations.versions = {
       global.tesla_coil.mines[mine_unit_number] = tower_data
     end
   end,
+    ["1.2.15"] = function()
```

Comment:
This actually isn't needed, `migrations.generic()` fires every time any mod updates.

Reply:
oh, it looked like [that if](https://github.com/raiguard/krastorio-2/blob/4f26ab30e0c0ed5de6fd4b4b1536508d3020c978/control.lua#L59) only triggered if the version was a key in this array here, are you absolutely sure? 🤔